site/docs/ch/flink.md

```diff
@@ -113,29 +115,33 @@ SHOW DATABASES;
 ```sql
 SHOW TABLES;
 ```
-## <a name="log">LOG 实时数据</a>
-**todo**：添加链接，format
+## LOG 实时数据<a name="log"></a>
 
```

Comment:
I prefer adding a link to the architecture introduction of Arctic here to make log-store more clear to users.